source/Halibut/Diagnostics/LogCreators/InMemoryConnectionLogCreator.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using Halibut.Logging;
+
+namespace Halibut.Diagnostics.LogCreators
+{
+    /// <summary>
```

Comment:
Nit: I feel this whole summary is unrequired, and a little confusing when it talks about caching.

Personally, I'd delete it, as I think `ICreateNewILog` implies it's going to create a new one with each call. And the caching thing is a little presumptuous on how we're going to use it.

Reply:
Improved the message here.